### ISSUE TITLE
Multiple right clicks open multiple context menus

### ISSUE THREAD

**Issue by tomas-correia** [PhaserEditor2D/PhaserEditor2D-v3#296]
## Version

* Phaser Editor 2D Version: 3.63.0

* Operating System: Windows 11

* Web Browser and version: Firefox Developer Edition, but I'm using the desktop version of Phaser Editor

## Description

On every right click, a new context menu appears. What should happen is on right-click, a context menu appears, and if a previous one exists, it should disappear.

Currently, the only way to hide previous context menus is to click one of its options, which might trigger undesired actions.

## Possible solution

Make previously created context menu disappear.

**Comment by PhaserEditor2D** [comment]
Yes, it happens to me too from time to time but I can't reproduce it at will. Please, if you find the pattern to reproduce tell me!

**Comment by tomas-correia** [comment]
Right now it's happening on every right-click for me. Another way to close the menu is by pressing ``Esc`` it seems.

**Comment by PhaserEditor2D** [comment]
Please, close the app and open it again and try to reproduce the issue. Then tell me how to reproduce it.

**Comment by PhaserEditor2D** [comment]
I fixed this in a previous version of the editor.
